src/App.js

```diff
@@ -107,134 +109,199 @@ export default class App extends React.Component<{}, AppState> {
             'forward1',
             'moveCharacter',
             () => {
-                this.audioManager.playSound('forward1');
+                // TODO: Enable announcements again.
+                // this.audioManager.playAnnouncement('forward1');
+
                 this.setState((state) => {
+                    const newCharacterState = state.characterState.forward(
```

Comment:
In the other command handlers below, you are placing the arguments to the new state call on one line, please update this call to be consistent.

Reply:
Fixed.